src/tactic/ui/tools/ingest_wdg.py

```diff
@@ -1666,6 +1667,11 @@ def execute(my):
 
             # Create a new entry
             if not sobject:
+                if update_mode not in ["true", True]:
+                    sobjects = []
+
```

Comment:
this whole if statement is for? the sobjects variable doesn't seem to be used any more past this point